documentation/modules/ROOT/pages/integration-with-aap.adoc

```diff
@@ -0,0 +1,176 @@
+
+[#workflow-overview]
+== Workflow Overview
+
+Now that we are familiar with AAP in OpenShift and the necessary components for integration, let's walk through an Ansible integration workflow.
+
+[#general-workflow]
+=== General AAP Integration Workflow
+
+At a high level, the steps to AAP integration with 5G RAN ZTP deployments are as follows:
+
+* Mirror AAP operator to disconnected registry (outside the scope of this lab, see the _Ansible Automation Platform mirroring_ section of this link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[blog])
+* Install AAP operator, deploy AutomationHub and AutomationController (already running on OpenShift cluster, see the _Ansible Automation Platform operator installation_ section of this link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[blog] and the CRD examples for link:integration-with-aap.html#automation-hub[AutomationHub] and link:integration-with-aap.html#automation-controller[AutomationController])
+* Upload necessary Ansible collections to Automation Hub (already uploaded, see uploaded collections at automation-hub-aap.apps.hub.5g-deployment.lab and instructions for this process in the link:lab-environment.html#upload-collections-to-aap-hub[Lab Environment] section)
+* Configure Automation Controller with Ansible automation resources
+* Install MultiClusterHub add-ons
+* Deploy spoke clusters
+* Deploy ManagedServiceAccount, Policy, PlacementBinding, PlacementRule, and PolicyAutomation for a governance-based hook into AAP
+* Spoke cluster(s) reach state where Policy applies, Ansible job is triggered by PolicyAutomation on each cluster, Ansible job completes successfully, Policy moves to compliant for that cluster
+
+This lab will cover the process starting with configuring the Automation Controller and ending with a successful Ansible job and a compliant automation Policy for a deployed cluster.
+
+[#lab-workflow]
+=== This Lab's AAP Integration Workflow
+
+This section will overview our example AAP integration workflow in this lab. If unfamiliar with the general AAP integration worflow, see the link:integration-with-aap.html#general-workflow[above section].
+
+In this lab, we will deploy a SNO cluster through the ZTP pipeline and await the completion of the SNO deployment, which will mark the SNO cluster with the _ztp-done_ label. The _ztp-done_ label will trigger an Ansible job that creates the namespace _ztp-day2-automation_ within our deployed SNO cluster. Assuming AAP is installed and deployed, and the necessary collections exist in our deployed Automation Hub, the steps to configure this integration is as follows:
+
+* *Configure Automation Controller with Ansible automation resources*
```

Comment:
As I put in the other docs, I think this is outside the scope of the lab and should be done in the automation. No need to explain the specific steps to the user since they will not be doing them.